modules/stack/variables.tf

```
@@ -256,3 +256,51 @@ variable "protect_from_deletion" {
   description = "Flag to enable/disable deletion protection."
   default     = false
 }
+
+variable "azure_devops" {
+  type        = map(any)
+  description = "(Block List, Max: 1) Azure DevOps VCS settings"
```

```suggestion
  description = "Azure DevOps VCS settings"
```

Remove `(Block List, Max: 1) ` from all vars, it's just confusing and does not explain anything about the var types (it's related to Terraform `dynamic` block)